vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt

```diff
@@ -147,13 +181,23 @@ class VectorJitsiActivity : VectorBaseActivity<ActivityJitsiBinding>(), JitsiMee
         super.onDestroy()
     }
 
-//    override fun onUserLeaveHint() {
-//        super.onUserLeaveHint()
-//        jitsiMeetView?.enterPictureInPicture()
-//    }
+    override fun onUserLeaveHint() {
+        super.onUserLeaveHint()
+        if (packageManager.hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE)) {
+            jitsiMeetView?.enterPictureInPicture()
```

Comment:
I think compat is already handled by Jitsi

Reply:
Yes, but better safe than sorry :)